main.py

```diff
@@ -0,0 +1,26 @@
+x = "I love Python" * 42
+print (x)
+
+age_in_month = 32 * 12
```

Comment:
It is better to change the value to exact amount of month, unless you have birthday in December